theano/configdefaults.py

```diff
@@ -1,11 +1,19 @@
+import errno
 import os
 import sys
 import logging
+import numpy
+import platform
+import textwrap
+import re
+import socket
 
 import theano
 from theano.configparser import (AddConfigVar, BoolParam, ConfigParam, EnumStr,
                                  FloatParam, IntParam, StrParam,
-                                 TheanoConfigParser)
+                                 TheanoConfigParser, THEANO_FLAGS_DICT)
+from theano.gof.compiledir import (local_bitwidth, python_int_bitwidth,
+                                   gcc_version_str)
```

Comment:
I did what you suggested, which solves this circular import but there seems to be many more circular import issues. With the last commit, I know have:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/adeb/code/Theano/theano/__init__.py", line 52, in <module>
    from theano.gof import (
  File "/Users/adeb/code/Theano/theano/gof/__init__.py", line 38, in <module>
    from theano.gof.cc import \
  File "/Users/adeb/code/Theano/theano/gof/cc.py", line 29, in <module>
    from theano.gof import graph
ImportError: cannot import name graph